engine/runner/src/main/resources/META-INF/native-image/org/enso/runner/serialization-config.json

```diff
@@ -284,27 +284,9 @@
   {
     "name":"org.enso.pkg.QualifiedName"
   },
-  {
-    "name":"org.enso.syntax.text.AST$ASTOf"
```

Comment:
I wonder: How did you find out about this file and that these lines should be removed? Did something fail?

Reply:
There was a warning that the file referred to a type that doesn't exist. Since I definitely deleted the type, I think whatever this line does, we don't need it